fvm/blueprints/scripts/systemChunkTransactionTemplate.cdc

```diff
@@ -1,9 +1,14 @@
 import FlowEpoch from 0xEPOCHADDRESS
+import NodeVersionBeacon from 0xNODEVERSIONBEACONADDRESS
 
 transaction {
   prepare(serviceAccount: AuthAccount) {
 	let heartbeat = serviceAccount.borrow<&FlowEpoch.Heartbeat>(from: FlowEpoch.heartbeatStoragePath)
       ?? panic("Could not borrow heartbeat from storage path")
     heartbeat.advanceBlock()
+
+    let versionBeacon = serviceAccount.borrow<&NodeVersionBeacon.NodeVersionAdmin>(from: NodeVersionBeacon.NodeVersionAdminStoragePath)
```

Comment:
The code indentation looks different than the `heartbeat` above, can we format the code properly?

Reply:
Not sure what you mean by "properly", we break line in the same place (before `??`) and indent in a same manner?

Reply:
I'm a bit nit here.
I meant the `	let heartbeat = ` used a tab as indentation, but the `    let versionBeacon = ` used 4 spaces as indentation. 
From github, it looks like the code is not formatted consistently:
![image](https://user-images.githubusercontent.com/811374/212392036-776beee0-a57b-47b8-90c8-1c49f224d90b.png)

Reply:
Interesting, it was showing the same in GoLand. Nevertheless, I have unified this with tabs everywhere and it seems consistent in github now.